docs/reference/esql/implicit-casting.asciidoc

```diff
@@ -31,35 +33,34 @@ FROM employees
 | LIMIT 1
 ----
 
-[float]
-=== Implicit casting support
+==== Operations that support implicit casting
 
 The following table details which {esql} operations support implicit casting for different data types.
 
 [%header.monospaced.styled,format=dsv,separator=|]
 |===
-||ScalarFunction*|Operator*|<<esql-group-functions, GroupingFunction>>|<<esql-agg-functions, AggregateFunction>>
-|DATE|Y|Y|Y|N
-|IP|Y|Y|Y|N
-|VERSION|Y|Y|Y|N
-|BOOLEAN|Y|Y|Y|N
-|DATE_PERIOD/TIME_DURATION|Y|N|Y|N
+|ScalarFunction*|Operator*|<<esql-group-functions, GroupingFunction>>|<<esql-agg-functions, AggregateFunction>>
```

Comment:
Instead of using  asterisk (`*`), I'd just use the plural (`s`); here and below, in the explanation.
Actually, I think I'd use the plural for all the categories in the header.